config/packages/dev/translation.yaml

```diff
@@ -0,0 +1,27 @@
+framework:
+    default_locale: '%locale%'
+    translator:
+        fallbacks:
+            - 'en'
+
+# php_translation
+translation:
+    locales: ['en', 'nl', 'es', 'fr', 'de', 'pl', 'it', 'pt_BR']
+    edit_in_place:
+        enabled: false
+        config_name: app
+    configs:
+        app:
+            dirs: ["%kernel.project_dir%/templates", "%kernel.project_dir%/src"]
+            output_dir: "%kernel.project_dir%/translations"
+            excluded_names: ["*TestCase.php", "*Test.php"]
```

Comment:
we don't need to exclude anything. tests are in separate dir, same with logs, cache.